From: Christian Melki <christian.melki@t2data.com>
Date: Fri, 28 Jan 2022 18:42:13 +0100
Subject: [PATCH] docs: Remove docs building.

efivar uses mandoc which ptxdist does not provide (yet).
Just skip the entire documentation generation and installation.

Signed-off-by: Christian Melki <christian.melki@t2data.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c896fc38c210..bf081a66970d 100644
--- a/Makefile
+++ b/Makefile
@@ -7,7 +7,7 @@ include $(TOPDIR)/src/include/defaults.mk
 include $(TOPDIR)/src/include/coverity.mk
 include $(TOPDIR)/src/include/scan-build.mk
 
-SUBDIRS := src docs
+SUBDIRS := src
 
 all : | efivar.spec src/include/version.mk prep
 all clean install prep :
